intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -21,7 +21,16 @@
 int main(int argc, char ** argv)
 {
   rclcpp::init(argc, argv);
-  auto camera_node = std::make_shared<CameraNode>("image");
-  rclcpp::spin(camera_node);
-  return 0;
+  try
+  {
+    auto camera_node = std::make_shared<CameraNode>("image");
+    rclcpp::spin(camera_node);
+    return 0;
+  }
+  catch( const std::exception& e )
```

Comment:
I think you need to run `ament_uncrustify` over these changes, I would expect the style to be:

```c++
} catch(const std::exception & e) {
```